src/identity/PublicIdentity.ts

```diff
@@ -15,6 +15,17 @@ export interface IURLResolver {
 }
 
 export default class PublicIdentity implements IPublicIdentity {
+  /**
+   * @description (STATIC) Creates a new Public Identity from a DID (Decentralised identifier) object.
+   * @param didDocument Public keys and pseudonymous biometrics
+   * @returns A new [[PublicIdentity]] object
+   * @example
+   * ```javascript
+   *
+   * PublicIdentity.fromDidDocument(didDocument);
```

Comment:
`fromDidDocument` has no side effects, so this example is not realistic. Instead you can do: 
```
const identity = PublicIdentity.fromDidDocument(didDocument);
```

Reply:
Nice, I have updated it.